packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -416,8 +416,13 @@ export class AtomicSearchBox {
         }`}
         onMouseDown={(e) => e.preventDefault()}
         onClick={() => suggestion.onSelect()}
+        ref={(e) => {
```

Comment:
I didn't quite understand what's going on with this! you said that we could get reference to the other children, how is that working here?

Reply:
For getting reference to the other children, that already exists, you can look at the `getSuggestions` method of the `SearchBoxSuggestionsBindings`. It is used in `AtomicSearchBoxQuerySuggestions` to check whether to add an icon or not